phantazm-mob/src/main/java/com/github/phantazmnetwork/mob/MobStore.java

```diff
@@ -0,0 +1,63 @@
+package com.github.phantazmnetwork.mob;
+
+import com.github.phantazmnetwork.mob.skill.Skill;
+import com.github.phantazmnetwork.mob.skill.SkillInstance;
+import net.kyori.adventure.key.Key;
+import net.minestom.server.entity.Entity;
+import net.minestom.server.event.entity.EntityDeathEvent;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.*;
+
+public class MobStore {
+
+    private final Map<UUID, PhantazmMob> uuidToMob = new HashMap<>();
+
+    private final Map<UUID, Map<Key, Collection<SkillInstance>>> uuidToTriggers = new HashMap<>();
+
+    public void useTrigger(@NotNull Entity entity, @NotNull Key key) {
+        UUID uuid = entity.getUuid();
+        PhantazmMob mob = uuidToMob.get(uuid);
```

Comment:
mob is unused, so containsKey more appropriate, unless you meant to use the mob somehow